specifyweb/specify/views.py

```diff
@@ -509,6 +509,32 @@ def fix_orderings(base_model: Table, new_record_data):
                     resources.insert(0, record)
             new_record_data[field_name] = resources
 
+def fix_record_data(new_record_data, model_name, old_model_ids, new_model_id):
+    final = {}
+    for key, value in new_record_data.items():
+
+        if isinstance(value, str) and model_name.lower() in value:
+            if "=" in value:
+                url = value.split("=")
+                if int(url[-1]) in old_model_ids:
+                    url[-1] = str(new_model_id)
+                final[key] = "=".join(url)
+                continue
```

Comment:
This isn't needed. Backend ignores to-many when putting.

Relevant lines https://github.com/specify/specify7/blob/038b7e52c6e72ad27850fa068021723dd1630208/specifyweb/specify/api.py#L552-L557